.circleci/ansible/provision.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name:
+    Provision the server, install build tools and dependencies
+    like Packer, VMware ovftool and Virtualbox.
```

Comment:
Same here.
While I know the context, would be good to sync the description up with the reality (remove ovftool mention), otherwise it might be confusing for others.